sdk/ai/azure-ai-projects/samples/agents/async_samples/sample_agents_stream_eventhandler_async.py

```diff
@@ -0,0 +1,102 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+FILE: sample_agents_stream_eventhandler_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to use agent operations with an event handler in streaming from
+    the Azure Agents service using a asynchronous client.
+
+USAGE:
+    python sample_agents_stream_eventhandler_async.py
+
+    Before running the sample:
+
+    pip install azure-ai-projects azure-identity
+
+    Set this environment variables with your own values:
+    PROJECT_CONNECTION_STRING - the Azure AI Project connection string, as found in your AI Studio Project.
+"""
+import asyncio
+from typing import Any
+
+from azure.ai.projects.aio import AIProjectClient
+from azure.ai.projects.models._models import (
+    MessageDeltaChunk,
+    MessageDeltaTextContent,
+    RunStep,
+    ThreadMessage,
+    ThreadRun,
+)
+from azure.ai.projects.models._patch import AsyncAgentEventHandler
```

Comment:
is the import from _patch proper ?

Reply:
Thanks @kashifkhan, good catch, the "._patch" part is not needed. Will be removed in my next PR.

Reply:
Removed